frontend/pages/hubs/[hubUrl]/index.tsx

```diff
@@ -0,0 +1,213 @@
+  try {
+    const resp = await apiRequest({
+      method: "get",
+      url: `/api/hubs/${hubUrl}/ambassador/`,
+      locale,
+    });
+    return resp.data;
+  } catch (err: any) {
+    console.error(
+      "Error fetching hub ambassador data:",
+      err.response?.data?.detail || err.message || err
+    );
+    return null;
+  }
+};
+
+const getHubData = async (url_slug: string, locale: LocaleType): Promise<HubData | null> => {
+  try {
+    const resp = await apiRequest({
+      method: "get",
+      url: `/api/hubs/${url_slug}/`,
+      locale,
+    });
+    return resp.data;
+  } catch (err: any) {
+    console.error("Error fetching hub data:", err.response?.data?.detail || err.message || err);
+    return null;
+  }
+};
+
+const LandingPage: React.FC<LandingPageProps> = ({ hubAmbassador, hubData, hubUrl }) => {
+  const { locale } = useContext(UserContext);
+  const texts = getTexts({ page: "landing_page", locale: locale }) as TextsType;
+  const [DevlinkComponent, setDevlinkComponent] = useState<DevlinkComponentType>(null);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+
+  useEffect(() => {
+    const loadComponent = async () => {
+      if (!hubData?.landing_page_component) {
+        setIsLoading(false);
+        return;
+      }
+
+      try {
+        // Identify the correct component name based on locale
+        let componentName = hubData.landing_page_component;
+        const currentPrefix = componentName.startsWith("En") ? "En" : "De";
+        const desiredPrefix = locale === "en" ? "En" : "De";
+
+        if (currentPrefix !== desiredPrefix) {
+          componentName = componentName.replace(new RegExp(`^${currentPrefix}`), desiredPrefix);
+        }
+
+        // Dynamically import the component
+        const mod = await import("../../../devlink");
+
+        if (mod[componentName]) {
+          setDevlinkComponent(() => mod[componentName]);
+        } else {
+          console.warn(`Component ${componentName} not found in devlink.`);
+          setDevlinkComponent(null);
+        }
+      } catch (error) {
+        console.error("Error loading devlink component:", error);
+        setDevlinkComponent(null);
+      }
+
+      // Set loading to false whether the try block succeeds or fails
+      setIsLoading(false);
+    };
+
+    loadComponent();
+  }, [locale, hubData]);
+
+  // Handle loading state
+  if (isLoading) {
+    return <LoadingSpinner isLoading color="#fff" noMarginTop />;
+  }
+  // Handle missing data
+  if (!hubUrl || !hubData) {
+    return <NotFoundPage texts={texts} link={buildHubUrl()} showHeader />;
+  }
+
+  const title = `${texts.climateHub} ${hubUrl} | ${texts.find_suitable_climate_protection_commitment} ${hubUrl}`;
+  const description = `${texts.find_fellow_campaigners_for_climate_protection_idea} ${
+    hubAmbassador?.title
+      ? `${hubAmbassador.title} ${texts.coordinates_the_climateHub}  ${hubUrl}  ${texts.is_there_for_you}`
+      : ""
```

Comment:
I would remove this from the description for now, I don't think it's really necessary